### DISCUSSION TITLE
Carattere voto lato docenti/genitori

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#107] [Idee e proposte]
Buongiorno, sarebbe possibile ingrandire il carattere dei voti ?Parlo sia per il lato docente che genitori/alunni .I voti risultano troppo minuscoli ,quasi invisibili,si vede solo quel quadratino verde rosso,arancione ,ripeto sia per il lato docente che genitore/alunno.L'unico modo per poter vederli bene dal lato genitore e cliccare sulla materia specifica,ma quando clicco su voti generali di tutte le materie non si vede quasi niente perché è troppo piccolo.Ed inoltre sarebbe possibile far modo che quando si passa sopra con il cursore si legga direttamente la descrizione,il tipo di voto,la data ,se fa media e il docente ?

**Comment by trinko** [comment]
Su uno schermo abbastanza grande non dovrebbero esserci problemi. Comunque il sito del registro è _responsive_, questo significa che si può ingrandire il contenuto di qualsiasi pagina utilizzando gli strumenti di ingrandimento (zoom) del proprio browser. 
Solitamente si usa la combinazione del tasto CTRL premuto assieme al + sul tastierino numerico.

Per la descrizione dei voti, c'è un problema tecnico: la possibilità di passare sopra un elemento con il mouse non esiste sui dispositivi mobili (tablet e smartphone), dove non si usa il mouse. Per questo è di uso più generale il click.

Lo segno comunque nelle richieste nel caso trovassi un'altra soluzione tecnica.
